webapp/views.py

```diff
@@ -32,6 +34,53 @@ def dashboard():
     return render_template("dashboard.jinja", groupings=groupings)
 
 
+@blueprint.route("/exams", methods=["GET"])
+def exams():
+    return render_template("exams.jinja")
+
+
+@blueprint.route("/exams", methods=["POST"])
+def enter_exam_group():
+    group = request.form['group'].upper()
+    print(group)
+    pattern = r"[\u0401\u0451\u0410-\u044f]{4}-[0-9]{2}-[0-9]{2}"
+    if re.match(pattern, group) != None:
```

Comment:
Заметим, что:

```python
>>> bool(None)
False
>>> bool(1)
True
```

Из чего следует, что данную строчку можно попробовать упростить до:

```suggestion
    if re.match(pattern, group):
```

Reply:
Исправил